syntax/std_seq_prefix_test.go

```diff
@@ -63,3 +63,65 @@ func TestBytesPrefix(t *testing.T) {
 	AssertCodesEvalToSameValue(t, `false`, `//seq.has_prefix(<<'o'>>, <<>>)    `)
 	AssertCodesEvalToSameValue(t, `true `, `//seq.has_prefix(<<>>, <<'hello'>>)`)
 }
+
+func TestStrTrimPrefix(t *testing.T) {
+	t.Parallel()
+	AssertCodesEvalToSameValue(t, `""`, `//seq.trim_prefix("ABC","ABC")`)
+
+	AssertCodesEvalToSameValue(t, `"BCDE"`, `//seq.trim_prefix("A","ABCDE")`)
+	AssertCodesEvalToSameValue(t, `"CDE"`, `//seq.trim_prefix("AB","ABCDE")`)
+	AssertCodesEvalToSameValue(t, `"ABCDE"`, `//seq.trim_prefix("BCD","ABCDE")`)
+	AssertCodesEvalToSameValue(t, `"ABCDE"`, `//seq.trim_prefix("CD","ABCDE")`)
+	AssertCodesEvalToSameValue(t, `""`, `//seq.trim_prefix("CD","")`)
+	AssertCodesEvalToSameValue(t, `"ABCD"`, `//seq.trim_prefix("","ABCD")`)
+
+	AssertCodesEvalToSameValue(t, `""`, `//seq.trim_prefix("","")`)
+	AssertCodesEvalToSameValue(t, `""`, `//seq.trim_prefix("A","")`)
+	AssertCodesEvalToSameValue(t, `"A"`, `//seq.trim_prefix("","A")`)
+
+	AssertCodeErrors(t, `//seq.trim_prefix(1,"ABC")`, "")
```

Comment:
what's the error message?

Reply:
actually it's a bit strange, the trim function doesn't seem to return error

Reply:
ah there's `AssertExprPanics`, but it's weird that `AssertCodeErrors` doesn't fail

Reply:
`wrapPanics` converts it to an error.

Reply:
I'm not a big fan of validating error messages, tbh. The important thing is that it fails.